sdks/cpp/build_scripts/msvs_2017_x64_release.bat

```diff
@@ -0,0 +1,22 @@
+@echo off
+REM Copyright 2018 Google Inc. All Rights Reserved.
+REM
+REM Licensed under the Apache License, Version 2.0 (the "License");
+REM you may not use this file except in compliance with the License.
+REM You may obtain a copy of the License at
+REM
+REM     http://www.apache.org/licenses/LICENSE-2.0
+REM
+REM Unless required by applicable law or agreed to in writing, software
+REM distributed under the License is distributed on an "AS IS" BASIS,
+REM WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+REM See the License for the specific language governing permissions and
+REM limitations under the License.
+
+pushd ..

```

Comment:
Don't know if this is a major issue, but these scripts need to be run from the build_scripts dir.
```
agones\sdks\cpp
$ build_scripts\msvs_2017_x64_release.bat
```
won't work

Reply:
I think it depends on personal preferences. Those scripts mostly for default build and demonstration of options. In real project there may be a lot of parent options like static/dynamic CRT, platform/compiler (x86, x64) etc.